Player/AppleTV/swift/AppleTV/ViewController.swift

```diff
@@ -11,19 +11,56 @@
 
 import BrightcovePlayerSDK
 
-let kViewControllerPlaybackServicePolicyKey = "BCpkADawqM3n0ImwKortQqSZCgJMcyVbb8lJVwt0z16UD0a_h8MpEYcHyKbM8CGOPxBRp0nfSVdfokXBrUu3Sso7Nujv3dnLo0JxC_lNXCl88O7NJ0PR0z2AprnJ_Lwnq7nTcy1GBUrQPr5e"
-let kViewControllerAccountID = "4800266849001"
-let kViewControllerVideoID = "5754208017001"
+fileprivate struct PlaybackConfig {
```

Comment:
Disclaimer: I do not know Swift style conventions. But, is it customary to capitalize the name of a struct var (as compared to a struct typedef)?

Reply:
Not sure - it just read nicer to me as a standalone thing. I could go either way with it.

Reply:
swift.org says to use lowerCamelCase (see "Follow case conventions" under "General Conventions"): https://swift.org/documentation/api-design-guidelines/#general-conventions

Reply:
Done